lib/view_backed.rb

```diff
@@ -19,7 +19,7 @@ def self.options
 
   class_methods do
     delegate :columns, to: :view_definition
-    prepend ViewBacked::Rails5 if Rails.version.match(/^5/)
+    prepend ViewBacked::Rails5 if Rails.version.match(/^(5|6)/)
```

Comment:
optional: I would consider renaming `ViewBacked::Rails5` to `ViewBacked::Rails5Or6` and `ColumnRails5` to `ColumnRails5Or6`.